boms/cloud-lts-bom/pom.xml

```diff
@@ -326,7 +323,6 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-tasks-bom</artifactId>
```

Comment:
@Neenu1995 Can I use this BOM to cover API surface?

Reply:
yes